module/VuFind/src/VuFind/ChannelProvider/ListItems.php

```diff
@@ -239,22 +239,11 @@ protected function getListsById($ids)
      */
     protected function addPublicLists($lists)
     {
+        $publicLists = [];
```

Comment:
This whole function could be further simplified to a ternary: `return $this->displayPublicLists ? array_merge($lists, $this->listService->getPublicLists()) : $lists;` -- and maybe that's a sign that we don't need the method at all now that you've refactored most of the logic to the database service.